pages/common/paperkey.md

```diff
@@ -0,0 +1,16 @@
+# paperkey
+
+> An OpenPGP key archiver.
+> More information: <https://www.jabberwocky.com/software/paperkey/>.
+
+- Take the secret key in `secret-key.gpg` and generate a text file `to-be-printed.txt` that contains the secret data:
+
+`paperkey --secret-key {{secret-key.gpg}} --output {{to-be-printed.txt}}`
```

Comment:
```suggestion
`paperkey --secret-key {{path/to/secret_key.gpg}} --output {{path/to/secret_data.txt}}`
```

Could you change the syntax to this.

Reply:
Done on c1db881.

Let me know if I should squash the commits together, I wasn't sure.

Reply:
We squash when merging, so it doesn't matter